CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ if(USE_CUDA)
     set(GPU_FFTLIB cufft)
     message(STATUS "CUDA generation enabled.")
     message(NOTICE "Using the available CUDA 'native' architecture.")
+    include_directories("${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES}")
```

Comment:
I think this will not cause any problems in general, and is probably something we should have included in the first place when compiling with CUDA support.